Makefile

```diff
@@ -104,6 +114,9 @@ build/esp32/: check-env
 	mkdir -p $@
 	make -C toolchains/esp32 -s $(shell pwd)/build/esp32/include/sdkconfig.h
 
+build/riscv64/:
```

Comment:
Merge this with the build/host/ line?

```
build/host/ build/riscv64/:
        mkdir -p $@
```

Reply:
This one I think we should keep separate for all cross-compile builds (also ARM/Linux, which I'm working on at the moment), since we'll soon need to use the host's toitc to generate the vm boot snapshot.

Reply:
Sounds good to me.